src/digit.rs

```diff
@@ -270,41 +275,6 @@ impl DoubleDigit {
     }
 }
 
-/// # Constructors
-impl Digit {
-    /// Creates a digit that represents the value `0`.
-    ///
-    /// **Note:** In twos-complement this means that all bits are `0`.
-    pub fn zero() -> Digit {
-        ZERO
-    }
-
-    /// Creates a digit that represents the value `1`.
-    pub fn one() -> Digit {
-        ONE
-    }
-
-    /// Returns `true` if this `Digit` is zero (`0`).
-    pub fn is_zero(self) -> bool {
-        self == ZERO
-    }
-
-    /// Returns `true` if this `Digit` is one (`1`).
-    pub fn is_one(self) -> bool {
-        self == ONE
-    }
```

Comment:
Why don't we keep those for readability?

Reply:
I'd like to keep the `is_one` and `is_zero` methods for readability. The others can be removed if you want.

Reply:
`is_zero` was just moved around in `digit.rs`